mithril-aggregator/src/file_uploaders/local_uploader.rs

```diff
@@ -144,10 +158,29 @@ mod tests {
         let uploader = LocalUploader::new(
             SanitizedUrlWithTrailingSlash::parse("http://test.com:8080/base-root/").unwrap(),
             &target_dir,
+            FileUploadRetryPolicy::never(),
             TestLogger::stdout(),
         );
         FileUploader::upload(&uploader, &source_dir)
             .await
             .expect_err("Uploading a directory should fail");
     }
+
+    #[tokio::test]
+    async fn retry_policy_from_file_uploader_trait_should_be_implemented() {
+        let target_dir = TempDir::create("local_uploader", "test_retry_policy");
+        let expected_policy = FileUploadRetryPolicy {
+            attempts: 10,
+            delay_between_attempts: Duration::from_millis(123),
+        };
+
+        let uploader: Box<dyn FileUploader> = Box::new(LocalUploader::new(
+            SanitizedUrlWithTrailingSlash::parse("http://test.com:8080/base-root/").unwrap(),
+            &target_dir,
+            expected_policy.clone(),
+            TestLogger::stdout(),
+        ));
```

Comment:
The use of `Box` doesn't seem necessary.
```suggestion
        let uploader = LocalUploader::new(
            SanitizedUrlWithTrailingSlash::parse("http://test.com:8080/base-root/").unwrap(),
            &target_dir,
            expected_policy.clone(),
            TestLogger::stdout(),
        );
```

Reply:
It's for have a `FileUploader` and not a `LocalUploader` to be sure that `retry_policy` is the function of the trait.
Otherwise, the test pass even if the trait use another function to retrieve policy. 
The test name try to explain that (`retry_policy_from_file_uploader_trait_should_be_implemented`) but it's probably not enough.
Do you think that we have to add a comment here to explain ?
// Cast into FileUploader to be sure it's the trait function that's being implemented

Reply:
Oh I get it, that sounds good to me like that 👍. The test name seems sufficient and clear to me.